apps/examples/wakaama_client/wakaama_client.c

```diff
@@ -394,6 +314,41 @@ int wakaamaclient_main(int argc, char *argv[])
 			printf("Read a resource value\n");
 			return 0;
 
+		case 'f': {
```

Comment:
Most of place checks below condition, but not here.
if (opt > argc) {
        app_print_usage();
        return 0;
}
Is doesn't neccesary here?
Also Is there any way improve this duplicated checking condition?